ApplicationLibCode/ProjectDataModel/RimWellTargetMapping.cpp

```diff
@@ -212,38 +224,43 @@ void RimWellTargetMapping::updateAllBoundaries()
     const int timeStepIdx = m_timeStep();
 
     auto updateBoundaryValues =
-        []( auto resultsData, const std::vector<RigEclipseResultAddress>& addresses, size_t timeStepIdx ) -> std::pair<double, double>
+        []( auto resultsData, const std::vector<RigEclipseResultAddress>& addresses, size_t timeStepIdx ) -> std::tuple<double, double, double>
     {
-        double globalMin = std::numeric_limits<double>::max();
-        double globalMax = -std::numeric_limits<double>::max();
+        double              globalMin = std::numeric_limits<double>::max();
+        double              globalMax = -std::numeric_limits<double>::max();
+        std::vector<double> allValues;
         for ( auto address : addresses )
         {
             double currentMinimum;
             double currentMaximum;
             if ( resultsData->ensureKnownResultLoaded( address ) )
             {
                 resultsData->minMaxCellScalarValues( address, timeStepIdx, currentMinimum, currentMaximum );
-                globalMin = std::min( globalMin, currentMinimum );
-                globalMax = std::max( globalMax, currentMaximum );
+
+                globalMin                         = std::min( globalMin, currentMinimum );
+                globalMax                         = std::max( globalMax, currentMaximum );
+                const std::vector<double>& values = resultsData->cellScalarResults( address, timeStepIdx );
+                allValues.insert( allValues.end(), values.begin(), values.end() );
             }
         }
-        return { globalMin, globalMax };
+
+        double p10, p50, p90, mean;
```

Comment:
It is not required to compute `P90,` this value is available using `RigCaseCellResultsData::p10p90CellScalarValues`

Reply:
No, I don't think that will work. We have multiple addresses here (e.g. TRANX, TRANY, TRANZ), and need the combined P90 of them.

Maybe (P90(TRANX) + P90(TRANY) + P90(TRANZ)) / 3.0 could be good enough, but it will be different that P90(TRANX + TRANY + TRANZ).